### ISSUE TITLE
Meet Intel FXC driver issue by #7748

### ISSUE THREAD

**Issue by qjia7** [tensorflow/tfjs#7783]
After #7748, some models in benchmarks got incorrect result on Intel driver with version `31.0.101.4369`. It seems like a regression on latest Intel drivers (bad: `31.0.101.4369`, good: `31.0.101.4032`).
With #7756, the first inference will get correct result. However, the following inferences still get incorrect result. It seems that second time goes into a different path in driver but gets wrong result.

After switching to dxc(`--enable-dawn-features=use_dxc`), all issues disappear.  So it looks like a fxc driver issue on intel device. Since webgpu in chrome will switch to dxc soon, record this issue here and won't fix.
